usingr.Rmd

```diff
@@ -0,0 +1,359 @@
+
+```{r}
+my_random_sum
+```
+
+With this, you have the very basics of how we write R code and save objects that can be used later.
+
+### Packages
+
+The base install of R is quite powerful, but you will soon have a need or desire to go beyond this.  Packages provide this ability.  They are a standardized method for extending R with new methods, techniques, and programming functionality.  There is a lot to say about packages regarding finding them, using them, etc., but for now let's focus just on the basics.  
+
+### CRAN
+
+One of the reasons for R's popularity is CRAN, [The Comprehensive R Archive Network](http://cran.r-project.org/).  This is where you download R and also where most will gain access to packages (there are other places, but that is for later).  Not much else to say about this now other than to be aware of it.  As of `r format(Sys.time(), "%Y-%m-%d")`, there are `r nrow(available.packages())` on CRAN! 
+
+### Installing packages
+
+When a package gets installed, that means the source is downloaded and put into your library.  A default library location is set for you so no need to worry about that.  In fact on Windows most of this is pretty automatic.  Let's give it a shot.
+
+## Exercise 2
+
+We're going to install some packages from CRAN that will give us the tools for our workshop today.  We'll use the tidyverse, sf, and mapview packages.  Later, we'll explain in detail what each of these packages provide.  
+
+1. At the top of the script you just created, type the following functions.
+
+    ```{r install_package, eval=FALSE}
+# install packages from CRAN
+install.packages("tidyverse")
+install.packages("sf")
+install.packages("mapview")
+    ```
+
+1. Select all three lines by clicking and dragging the mouse pointer over the text.  
+
+1. Send all threee commands to the console using `ctrl+enter`.  You should see some text output on the console about the installation process.  The installation may take a few minutes so don't be alarmed.  
+
+1. After the packages are done installing, verify that there were no errors during the process (this should be pretty obvious, i.e., error text in big scary red letters).
+
+1. Load the packages after they've installed. 
+
+    ```{r eval=FALSE}
+library("tidyverse")
+library("sf")
+library("mapview")
+    ```
+
+### Getting Help
+
+Being able to find help and interpret that help is probably one of the most important skills for learning a new language.  R is no different. Help on functions and packages can be accessed directly from R, can be found on CRAN and other official R resources, searched on Google, found on StackOverflow, or from any number of fantastic online resources. I will cover a few of these here. 
+
+### Help from the console
```

Comment:
Maybe add a link or mention the difference between "warnings" and "errors" and what each means (or how to use that information to google stuff).

Reply:
Good call, added 29d7eb968d6869b49c484e028c6ed4bc911b8c01